hal/targets/cmsis/TARGET_ARM_SSG/TARGET_BEETLE/system_CMSDK_BEETLE.c

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2009-2016 ARM Limited. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+ * This file is derivative of CMSIS V5.00 system_ARMCM3.c
+ */
+
+#include "cmsis.h"
+
+/*----------------------------------------------------------------------------
+ * Define clocks
+ *----------------------------------------------------------------------------*/
+#define __XTAL (48000000UL) /* Oscillator frequency */
+
+#define __SYSTEM_CLOCK (__XTAL / 2)
+
+/*----------------------------------------------------------------------------
+ * Clock Variable definitions
+ *----------------------------------------------------------------------------*/
+/* !< System Clock Frequency (Core Clock) */
+uint32_t SystemCoreClock = __SYSTEM_CLOCK;
+
+/*----------------------------------------------------------------------------
+ * Clock functions
+ *----------------------------------------------------------------------------*/
+/**
+ * Update SystemCoreClock variable
+ *
+ * @param  none
+ * @return none
+ *
+ * @brief   Updates the SystemCoreClock with current core Clock
+ *          retrieved from cpu registers.
+ */
+void SystemCoreClockUpdate (void)
+{
+
+    SystemCoreClock = __SYSTEM_CLOCK;
```

Comment:
As reported on top of the file "This file is derivative of CMSIS V5.00 system_ARMCM3.c". This is how it is implemented in the original file and reports the same comment.

MBED for Beetle at the moment supports only the default 24Mhz configuration so the implementation is coherent with the purpose.